extension.h

```diff
@@ -43,8 +43,8 @@ struct HTTPRequest {
 		}
 	}
 
-	const char *method;
-	const char *endpoint;
+	const ke::AString method;
+	const ke::AString endpoint;
 	json_t *data;
 
 	char *body;
```

Comment:
Didn't do this one as `ke::AString` currently has no way to adopt a `char *` buffer, and it could be quite a large copy.